### DISCUSSION TITLE
Bell (buzzer) question

### DISCUSSION THREAD

**Discussion by IMSAI-Oracle** [thehighnibble/thehighnibble.github.io#62] [S-132 add-on for IMSAI & Cromemco replicas]
Just getting ready to install my bell and noticed this small copper wire on the outside. Not sure they should be here, but I'm afraid to pull it apart and stick it inside.

UPDATE: However it works perfectly 👍 

![image](https://github.com/thehighnibble/thehighnibble.github.io/assets/131218791/2f9b5c5d-a0d6-4046-bfb4-464b6540d56b)

Thanks,
Lynn M.

**Comment by GrantMeStrength** [comment]
I don't see any wires like that on any of the piezo speakers I've connected.. I think it's just a random piece of stuff that was floating through the universe and liked your speaker.
